_Streams/114.1-neural-networks-continued.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Neural Networks Continued"
+video_number: 114.1
+date: 2018-01-12
+video_id: iCxqLVm2H3w
+
+topics:
+  - title: "Improving Matrix class"
+    time: "21:11"
+    url: "https://www.youtube.com/watch?v=HuZbYEn8AvY"
+  - title: "Feed Forward Neural Networks Pt.1"
+    time: "55:45"
+    url: "https://www.youtube.com/watch?v=qWK7yW8oS0I"
+  - title: "Feed Forward Neural Networks Pt.2"
+    time: "1:40:15"
+    url: "https://www.youtube.com/watch?v=MPmLWsHzPlU"
+  - title: "Coding Challenge: Dithering"
+    time: "2:21:14"
+    url: "/CodingChallenges/90-floyd-steinberg-dithering"
+ 
+links:
+  - title: "The Coding Train on Amazon"
+    url: "https://www.amazon.com/shop/thecodingtrain"
+  - title: "Deeplearn.js"
+    url: "https://deeplearnjs.org/"
+  - title: "Repository for Experimentation with p5 and deeplearn.js"
+    url: "https://github.com/ITPNYU/ml5"
+  - title: "Dither on Wikipedia"
+    url: "https://en.wikipedia.org/wiki/Dither"
+  - title: "Floyd–Steinberg dithering on Wikipedia"
+    url: "https://en.wikipedia.org/wiki/Floyd%E2%80%93Steinberg_dithering"
+  - title: "Sigmoid function on Wikipedia"
+    url: "https://en.wikipedia.org/wiki/Sigmoid_function"
+  - title: "Stippling by Robert Hodgin"
+    url: "http://roberthodgin.com/portfolio/stippling/"
+  - title: "My Processing Tutorial on 2D Arrays"
+    url: "https://processing.org/tutorials/2darray/"
+  - title: "Matrix Multiplication"
+    url: "http://matrixmultiplication.xyz/"
+  - title: "Static methods on MDN"
+    url: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static"
+
+videos:
+  - title: "My Neural Networks series"
+    url: "https://www.youtube.com/playlist?list=PLRqwX-V7Uu6aCibgK1PTWWu9by6XFdCfh"
+  - title: "3Blue1Brown Neural Networks playlist"
+    url: "https://www.youtube.com/playlist?list=PLZHQObOWTQDNU6R1_67000Dx_ZCJB-3pi"
+  - title: "3Blue1Brown's Linear Algebra playlist"
+    url: "https://www.youtube.com/playlist?list=PLZHQObOWTQDPD3MizzM2xVFitgF8hE_ab"
+  - title: "My Tutorial on Pixels in Processing"
+    url: "https://www.youtube.com/watch?v=EmtU0eloTlE"
+  - title: "My Video on 2D Arrays"
+    url: "https://www.youtube.com/watch?v=OTNpiLUSiB4"
```

Comment:
This video is already available on the website. Could you please use the internal URL?